Dockerfile

```diff
@@ -24,7 +24,9 @@ RUN useradd --create-home --shell /bin/bash $USER && \
     chown -R $USER /home/$USER && \
     # install server components
     apt-get update && \
-    apt-get install -qq --no-install-recommends gettext nginx
+    apt-get install -qq --no-install-recommends gettext nginx && \
+    # install git for use when installing dependencies from VCS (https://pip.pypa.io/en/stable/topics/vcs-support/)
+    apt-get install -qq --no-install-recommends git
```

Comment:
Quick notes: Installing git here is necessary to use pip's [VCS support](https://pip.pypa.io/en/stable/topics/vcs-support/) in `requirements.txt` (see [line 5](https://github.com/cal-itp/benefits/pull/606/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R5) in `requirements.txt`).

I don't expect this line to be around long-term, so I wanted to separate it out and make it stick out so that it doesn't look like a long-term change. 

If there are any strong feelings on this though, let me know! (For example, maybe you'd rather we just put `git` at the end of line 27. Or maybe the comment should be more explicit about this being temporary.)

Reply:
Thanks for the reasoning around keeping it on a separate line, I was going to comment and ask about that.

Maybe just create an issue to follow-up and remove this when you consume the PyPI package.

Reply:
Created #612 for this and put it in the "Blocked" column since it requires cal-itp/eligibility-api#22 to be done.